vcd/datasource_vcd_org.go

```diff
@@ -0,0 +1,78 @@
+package vcd
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func datasourceVcdOrg() *schema.Resource {
+	return &schema.Resource{
+		Read: datasourceVcdOrgRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Organization name for lookup",
+			},
+			"full_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
All other attributes should be Computed, not optional and forcenew. Cause read is done only by name

Reply:
done